src/context/WalletProvider/TallyHo/components/Connect.tsx

```diff
@@ -74,10 +74,7 @@ export const TallyHoConnect = ({ history }: TallyHoSetupProps) => {
         // Switch to Mainnet if wallet is on any other chain
         const chainId = await wallet.ethGetChainId?.()
         if (bnOrZero(chainId).toString() !== CHAIN_REFERENCE.EthereumMainnet) {
```

Comment:
Good question, I think it doesn't matter which one we go for currently.

I reverted this hunk to make it consistent in this PR that wallets that explicitly `supportsEvmSwitchChain()` don't have the "Unsupported wallet" copy anymore, while the ones that don't (which is the case for TallyHo, see https://github.com/shapeshift/hdwallet/pull/547/files#diff-0fd42be91787531dbf00ceccd748f244eda06e374a6cddc3ef0f652569235afbR86) keep the old logic.

When we implement multi-chain for Tally, it will be two PRs instead of one, but at least we're properly honoring the `supoprtsEvmSwitchChain` logic

Reply:
cool, sounds like you thought it through, just making sure wasn't accident. thanks for the detailed explanation as usual! :)